tools/resource-generator/resource-generator.go

```diff
@@ -56,14 +56,15 @@ func newKubeVirtCR(namespace string, pullPolicy v1.PullPolicy, featureGates stri
 		},
 		Spec: virtv1.KubeVirtSpec{
 			ImagePullPolicy: pullPolicy,
+			Configuration: virtv1.KubeVirtConfiguration{
+				ImagePullPolicy: pullPolicy,
+			},
```

Comment:
```suggestion
			Configuration: virtv1.KubeVirtConfiguration{
				ImagePullPolicy: pullPolicy,
			},
```

Reply:
> Yeah AFAICT it's not, just the resource-generator and a random unit test. Definitely something to clearly deprecate and document in a follow up (that I'm happy to do).

Here is where they are used:
https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-operator/util/config.go#L69-L88

Reply:
Wow, that's fun. So I guess this stays after all.